export/imagepipeline.py

```diff
@@ -65,6 +65,9 @@ def convert_defs(context, scene, definitions, plugin_index, define_radiancescale
         if pipeline.contour_lines.enabled:
             index = _contour_lines(definitions, index, pipeline.contour_lines)
 
+        if pipeline.white_balance.enabled:
+            index = _white_balance(definitions, index, pipeline.white_balance)
```

Comment:
I think I would put the white balance plugin after the vignetting plugin and in front of the camera response func. (Not sure about the latter though, maybe it should go after the camera response?).

Reply:
So, the plugins are currently applied in a fixed order?
Don't we have a node panel for that too?

I think we might need one to support [this](http://forums.luxcorerender.org/viewtopic.php?f=5&t=1017&p=17360#p17358)

Reply:
Yes, currently the order is fixed. I thought about making it customizable, but so far it was not necessary (there was only one order that made sense).
If we make it customizable, I think the node editor would be overkill because it's only image in -> image out. Simple "up/down" buttons near the plugins would be better I think.

Reply:
That would work.

Do you think we can have multiple possible pipelines without the nodes?
I have already uncoupled the convergence test and noise estimation from the first image pipeline, so my initial idea is to generate a second image pipeline that would mimic the setting in Blender, so the adaptive sampling works as intended.

If that works nicely, we can then decouple the "reference" image and the one used for calculating and have the adaptive sampling driven by arbitrary inputs, such as the denoise channel suggested by @Dade916 on the forums

Reply:
Do these additional pipelines have to be accesible from the UI?
Because it sounds to me like they don't have to be in the UI?
And i'm already creating such pipelines, e.g. for the light groups.

Reply:
For the denoise, no.
If we wanted to give more options, maybe.

Let's keep it easy and just export in the background for now. I am still struggling to get OCIO and OIIO to compile here